src/libecalc/common/string_utils.py

```diff
@@ -34,4 +34,5 @@ def to_camel_case(string: str) -> str:
 
     """
     string_split = string.replace("__", "_").split("_")
+    string_split = [word for word in string_split if len(word) > 0]  # Allow names such as 'from_'
```

Comment:
Allow words that end with an underscore?

Reply:
ends and starts I think, I added the commented test cases. Needed for alias when having an attribute that is a reserved word in python